docker/Dockerfile

```diff
@@ -1,6 +1,21 @@
-FROM circleci/openjdk:8-jdk
+FROM maven:3-jdk-11-slim
 
-COPY . /code
-WORKDIR /code
+ADD . /code
 
-ENTRYPOINT ["./docker/run.sh"]
+RUN cd /code && mvn -B --no-transfer-progress package -DskipTests=true
+
+FROM gcr.io/distroless/java:11-debug
+
+RUN ["/busybox/sh", "-c", "adduser -g '' -h /zipkin -D zipkin"]
```

Comment:
not in love with the username and work dir as it looks more like what's used in zipkin, and this isn't zipkin. however, I'm lacking another option and don't really care :P